readme.md

```diff
@@ -81,6 +81,7 @@
 ## IDE
 
 - [swish](https://github.com/SWI-Prolog/swish) - SWI-Prolog web IDE.
+- [Prolog extension](https://github.com/arthwang/vsc-prolog) for [Visual Studio Code](https://code.visualstudio.com/).
```

Comment:
From the [contributing guidelines](https://github.com/sindresorhus/awesome-nodejs/blob/master/contributing.md):

> - Use the following format: [package](link) - Description.
> - Additions should be added to the bottom of the relevant category.

Also, something like following would be more fitting: 

- [VSC-Prolog](https://github.com/arthwang/vsc-prolog) -  Prolog extension Visual Studio Code.

Reply:
Fixed.